week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,9 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = students.every(function(student) {
+  return group.includes(student) !== true;
```

Comment:
I think you had a good idea, but what you're doing here is that you're checking if all students are included in the `group` array. What you need to check is if the `group` array contains only students - subtle difference :D